swatch-core/src/main/java/org/candlepin/subscriptions/task/TaskQueueProperties.java

```diff
@@ -41,4 +41,8 @@ public class TaskQueueProperties {
 
   /** Batch size number of records * */
   private String maxPollRecords = "500";
+
+  private int retryAttempts = 2;
+
+  private long retryBackOffMillis = 5000L;
```

Comment:
Since we will retry only once may be we can remove retryBackOffMillis?
Also I assume the default retention of a Kafka topic is one week. I am wondering if for DLT we should reduce it if not consumed?

Reply:
I set the backoff to 0 just in case we want to override later. One week sounds ok to me for it.